client_taxi_driver/shared/src/commonMain/kotlin/presentation/composable/noRippleEffect.kt

```diff
@@ -0,0 +1,14 @@
+package presentation.composable
+
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.interaction.MutableInteractionSource
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+
+@Composable
+fun Modifier.noRippleEffect(onClick: () -> Unit) = clickable(
+    interactionSource = remember { MutableInteractionSource() }, indication = null
+) {
+    onClick()
+}
```

Comment:
why we want to remove the ripple ? we can change it's color or how it's look but it's a good practice to have a ripple effect on click

Reply:
done